### ISSUE TITLE
option to use server time for cases

### ISSUE THREAD

**Issue by jmtaylor90** [CERT-BDF/TheHive#260]
### Request Type
Feature Request

### Work Environment

| Question              | Answer
|---------------------------|--------------------
| OS version (server)       | CentOS
| OS version (client)       | Any
| TheHive version / git hash   | 2.12.0, hash of the commit
| Package Type              | Binary
| Browser type & version    | Any

### Problem Description
When creating a new case it appears to use the client time for the case entry. It would be nice to have an option to allow the cases to use the server time. In our case we use UTC for our work but a workstation/client may be set to a different time zone.

**Comment by nadouani** [comment]
I agree, but this would require adding support to user specified time zones. The ideal would to allow the user to choose the timezone he wants, use that timezone to display the dates, and make sure all the dates are stored on UTC on the server.

I have to double check how the dates are store currently, but it's most certainly in UTC.

**Comment by architect00** [comment]
May I ask about the current progress on this issue?

We are currently reconfiguring all our servers to UTC and also look for an option to have a unified display format.
